### ISSUE TITLE
Token name overlaps icon in Token Type dropdown

### ISSUE THREAD

**Issue by emergent-reasons** [simpleledger/Electron-Cash-SLP#67]
I do not see the same issue at current head 57492a9f but just making sure the fix is intentional.

Having the dropdown open disables standard screenshotting but I can get a picture if you need one.

OS: Mint 19
Kernel: 4.15
Electron: SLP 3.4.10 AppImage

**Comment by jcramer** [comment]
same issue exists in dark mode

**Comment by emergent-reasons** [comment]
> same issue exists in dark mode

Yes! Actually I am using dark mode. I did not check in the normal theme/UI.

But it seems to be fixed already in 3.4.11?

**Comment by jcramer** [comment]
Its still there I think in latest release 3.4.12, Calin was looking at it today and he thinks it may be a problem with the dark theme dependency

**Comment by cculianu** [comment]
@emergent-reasons  Well it's fixed on Linux now, which is good.  I am not sure why it was happening before and/or what fixed it but it's fixed on Linux. (I didn't spend time investigating what fixed it -- a bunch of things changed simultaneously including patches to QDarkStyle).

It's still happening on macOS only as far as I can tell.  On Linux & Windows it appears fixed.

It's definitely a bug with QDarkStyle's style -- I went in and read over the whole stylesheet and I can't quite figure out which of the things they are doing would trigger it -- it all seemed relatively kosher stuff.

I submitted a bug report to the QDarkStyle project -- they are usually pretty responsive, actually.  Here is the issue on their project: https://github.com/ColinDuquesnoy/QDarkStyleSheet/issues/169
